packages/api/tests/models/test-collections-model.js

```diff
@@ -100,13 +87,31 @@ test('Collection.delete() throws an exception if the collection has associated r
 test('Collection.delete() deletes a collection', async (t) => {
```

Comment:
Minor quibble: Gven we're now deleting from both the database and the config store, should we update the title of this test to be explicit/document the intended behavior, e.g. "Collection.delete() deletes a collection and removes it's configuration store" ?

Reply:
Yep, fair enough. Would you like me to do that before accepting this PR?

Reply:
Yeah, this and the comment in https://github.com/nasa/cumulus/pull/1141#discussion_r311822508 were the two I was referring to.    Apologies for not just including links in the first place.